README.md

```diff
@@ -257,6 +332,24 @@ End with agreements:
 
 One of the quickest ways to derail honest communication is by presuming to know the intentions of the other person. Remember to "stay on your side of the net"—focus on specific, observable behavior and ask the other person about their motives and intentions.
 
+## Being a point person :point_up:
+
+Point people are our guides to help us define our roles and set and meet goals on an ongoing basis. Everyone in the company has a point person—even the CEO. Point people are not managers, and that means they people have a unique set of responsibilities at Able Health. These include:
+
+* Working with team members to define roles and set goals in January and July
+
+* Meeting one-on-one with team members on a regular basis
+
+* Supporting team members in achieving their goals
+
+* Working with team members to reflect on and update roles
+
+* Helping team members navigate conflicts related to roles and goals
+
+Point people _do not_ have the inherent authority to assign work or make decisions related to compensation, promotion, demotion, or firing.
```

Comment:
Compensation will still be decided by the founders since we don't have a formal budget allocation process. We can implement a promotion mechanism around a broader band level/career ladder system as you have described for engineering. I don't think it's urgent given the size and makeup of our current team, but I will seek advice about this and we can put something in place when we start hiring more junior folks.